src/patternfly/components/DualListSelector/dual-list-selector-item.hbs

```diff
@@ -1,7 +1,12 @@
 {{#> dual-list-selector-list-item-row}}
-  <{{#if dual-list-selector-item--type}}{{dual-list-selector-item--type}}{{else}}button{{/if}} class="pf-c-dual-list-selector__item{{#if dual-list-selector-item--modifier}} {{dual-list-selector-item--modifier}}{{/if}}"
-    {{#if dual-list-selector-item--type}}
-      tabindex="0"
+  <{{#if dual-list-selector-item--type}}{{dual-list-selector-item--type}}{{else}}button{{/if}} class="pf-c-dual-list-selector__item{{#if dual-list-selector-item--IsDisabled}} pf-m-disabled{{/if}}{{#if dual-list-selector-item--modifier}} {{dual-list-selector-item--modifier}}{{/if}}"
```

Comment:
For this block I think you want to add some checks around `pf-m-disabled` and the `disabled` attr. A button should have `disabled` and a div/non-button should have `pf-m-disabled`.

```
<{{#if dual-list-selector-item--type}}{{dual-list-selector-item--type}}{{else}}button{{/if}} class="pf-c-dual-list-selector__item
  {{~#if dual-list-selector-item--IsDisabled}}
    {{~#if dual-list-selector-item--type}} pf-m-disabled{{/if}} // add pf-m-disabled if it's a disabled div/non-button
  {{/if}}
  {{~#if dual-list-selector-item--modifier}} {{dual-list-selector-item--modifier}}{{/if}}"
  {{#unless dual-list-selector-item--IsDisabled}}
    {{#if dual-list-selector-item--type}}
      tabindex="0"
    {{/if}}
  {{/unless}}
  {{#if dual-list-selector-item--IsDisabled}}
    {{#unless dual-list-selector-item--type}}
      disabled // add disabled attr if it's a disabled button
```